modules/godot/editor_plugins/editor_world_ecs.cpp

```diff
@@ -1,3 +1,4 @@
+#ifdef TOOLS_ENABLED
```

Comment:
This `ifdef` is surrounding the entire file, in theory the `ifdef` inside the file can be removed, what do you think?

Reply:
You are right, I did not notice the other ifdef.